### DISCUSSION TITLE
How to avoid duplicate card of same word?

### DISCUSSION THREAD

**Discussion by ghost** [FreeLanguageTools/vocabsieve#139] [Q&A]
I checked there were an issue and u said I recommend users should use word as first field I don't know, I did something but did not work I am new to anki. I changed note type a bit so I am sending it maybe it would help to reproduce?
https://we.tl/t-E7PWv8zXos

**Comment by 1over137** [comment]
Describe your problem in more detail. What exactly did you do? What isn't working? The default note type should have word as first field.

**Reply by ghost** [reply]
I create cards with vocabsieve, but most of the time I have already created another card for that word. I can't avoid this. this is my goal.
current card formatting.
front
```
<div class="widget front">

  <div class="title">What does this word mean here?</div>
  <div class="box targetLang"> 
		  {{#Word}}<div class="box targetLang"> 
    <div class="question"><b>{{Word}}</b></div>
  </div>{{/Word}}
    <div class="question">{{Audio}} {{Sentence}}</div>
  </div>



</div>
```
back
```
{{FrontSide}}

<hr id=answer>

<div class="widget back">
		
		{{#Definition}}<div class="box nativeLang answer">{{Definition}}</div>{{/Definition}}

		{{#Definition#2}}<div class="box nativeLang answer">{{Definition#2}}</div>{{/Definition#2}}
	
	{{#Image}}<div class="image">{{Image}}</div>{{/Image}}
	
		{{#Pronunciation}}<div class="box targetLang">{{Pronunciation}}</div>{{/Pronunciation}}

{{#Tags}}<div class="tags">{{Tags}}</div>{{/Tags}}

</div>
```
but still can have duplicate cards
![image](https://github.com/FreeLanguageTools/vocabsieve/assets/43301560/18806f9a-b552-4e9b-912c-70fe5e515162)
![image](https://github.com/FreeLanguageTools/vocabsieve/assets/43301560/6a677221-052b-48b8-ad67-c829b95e980f)
![image](https://github.com/FreeLanguageTools/vocabsieve/assets/43301560/b1147d9a-b295-45b1-aebb-c9d2dc891494)
![image](https://github.com/FreeLanguageTools/vocabsieve/assets/43301560/6519fe50-f501-4023-bfba-fd795620a44d)

audio is from mpvacious pronouncation is from vocabsieve so I want to create 1 card each word. I dont wanna have duplicate cards

**Reply by 1over137** [reply]
Formatting doesn't matter here, only the fields in note type. Are they the same note types? Anki only detects duplicates when they are of the same note type. You may need to convert them.

**Reply by ghost** [reply]
yeah they are using same note type I created them with these settings.
![image](https://github.com/FreeLanguageTools/vocabsieve/assets/43301560/16f8bc6e-fe78-4ada-b2c7-7cc8eea9a504)

**Reply by ghost** [reply]
ohhh I figured it out rn changed field numnber from fields settings I had changed formatting before sorry for nonsense
